libs/orders/src/lib/uitls.ts

```diff
@@ -0,0 +1,22 @@
+import { t } from '@vegaprotocol/react-helpers';
+import { OrderTimeInForce } from '@vegaprotocol/types';
+
+// More detail in https://docs.vega.xyz/docs/mainnet/graphql/enums/order-time-in-force
```

Comment:
praise (non-blocking): 💯 
chore (blocking):  there's a typo in the name. please rename file to utils.ts